src/ir-util.cc

```diff
@@ -260,4 +259,8 @@ ModuleContext::Arities ModuleContext::GetExprArity(const Expr& expr) const {
     case ExprType::SimdShuffleOp:
       return { 2, 1 };
   }
+
+  fprintf(stderr, "bad expr type: %s\n", GetExprTypeName(expr));
```

Comment:
Is it that this case is fully covered?   In that case I would use something like LLVM_UNREACHABLE here.

Reply:
Yeah, good point.